apps/editing-toolkit/editing-toolkit-plugin/editor-site-launch/src/launch-steps/final-step/index.tsx

```diff
@@ -33,14 +33,31 @@ import './styles.scss';
 const TickIcon = <Icon icon={ check } size={ 17 } />;
 
 const FinalStep: React.FunctionComponent< LaunchStepProps > = ( { onNextStep, onPrevStep } ) => {
-	const domain = useSelect( ( select ) => select( LAUNCH_STORE ).getSelectedDomain() );
-	const plan = useSelect( ( select ) => select( LAUNCH_STORE ).getSelectedPlan() );
 	const planPrices = useSelect( ( select ) =>
 		select( PLANS_STORE ).getPrices( window.wpcomEditorSiteLaunch?.locale || 'en' )
 	);
-	const LaunchStep = useSelect( ( select ) => select( LAUNCH_STORE ).getLaunchStep() );
-	const isStepCompleted = useSelect( ( select ) => select( LAUNCH_STORE ).isStepCompleted );
-	const isFlowCompleted = useSelect( ( select ) => select( LAUNCH_STORE ).isFlowCompleted() );
+
+	const { domain, plan, LaunchStep, isStepCompleted, isFlowCompleted, planProductId } = useSelect(
+		( select ) => {
+			const launchStore = select( LAUNCH_STORE );
+			return {
+				domain: launchStore.getSelectedDomain(),
+				plan: launchStore.getSelectedPlan(),
+				LaunchStep: launchStore.getLaunchStep(),
+				isStepCompleted: launchStore.isStepCompleted,
+				isFlowCompleted: launchStore.isFlowCompleted(),
+				planProductId: launchStore.getSelectedPlanProductId(),
+			};
+		}
+	);
+
+	const { planProduct, isPlanFree } = useSelect( ( select ) => {
+		const plansStore = select( PLANS_STORE );
+		return {
+			planProduct: plansStore.getPlanProductById( planProductId ),
+			isPlanFree: plansStore.isPlanProductFree( planProductId ),
+		};
+	} );
```

Comment:
`Plan.isFree` is still available. So this should do: 

```suggestion
	const planProduct = useSelect( ( select ) =>
		select( PLANS_STORE ).getPlanProductById( planProductId )
	);
```